R/dd_to_dms.R

```diff
@@ -0,0 +1,30 @@
+
+#' dd_to_dms
+#'
+#' @description  Converts decimal degrees to degrees, minutes, and seconds (DMS) format.
+#'
+#' @param x Numeric value representing the decimal degree to be converted.
+#' @param lat Logical value indicating whether the conversion is for latitude (TRUE) or longitude (FALSE).
```

Comment:
For TRUE and FALSE, can you put ` < this symbol around the words so it goes into code format, like:

`TRUE` and `FALSE`

Reply:
> 

Tried to make the changes but not sure if that's what you meant. Review and let me know if i need to do improve on it.